functions/src/import-votes.ts

```diff
@@ -8,10 +8,10 @@ const config = functions.config() as Config;
 const importVotesConfigs = config.features.import_votes;
 const db = firebase.firestore();
 
-const fromJSONtoVote = supposedlyValidVotes => {
+const fromJSONtoVote = (supposedlyValidVotes: any) => {
```

Comment:
Si je ne dis pas de bêtises ça ne sert à rien de mettre any de toute manière l'inférence de type va s'appliquer ?

Reply:
This `noImplicitAny` compiler option is enabled on this project so it needs to be explicit. However I don't think this parameter should have the `any` type. I have to keep on reading to make sure.

Reply:
The parameter type should be `unknown` to maximize type safety. `any` makes anything pass, while `unknown` makes everything fail, which force the code to check everything.

Reply:
how should it check it ?

Reply:
With `typeof`.

Reply:
Let's say this is out of scope of this PR. This PR is just quick fixing it to make it compile.